docs/knowledge_base/tips/multipage_application.md

```diff
@@ -0,0 +1,191 @@
+![Defining the Pages](multipage_application_5.png){width=100%}
+
+## Defining a Multi-Page Application
+
+Up to this point, we've structured our multi-page Taipy application by 
+keeping two one-page applications in a subfolder named "pages."
+
+Now, the next step is to create and define our main module, `main.py` within the *app* directory. 
+After that, we'll initialize our multi-page Gui object.
+
+![Defining a Multi-Page Application](multipage_application_6.png){width=100%}
+
+We started by importing two Markdown objects, *home_md* and *temperature_md* from the two scripts we previously created. 
+Then, we made a dictionary called `pages`:
+
+1. Each key in the dictionary specifies the URL where you can access that page.
+2. Each value in the dictionary is the *Markdown* object for that page.
+
+In the end, we provided this *pages* dictionary as an argument to the *pages* parameter of the Taipy Gui object. 
+Then, we called its *run* method, and that's how we got our first Taipy multi-page application working!
+
+If you open your web browser and go to **localhost:5000** (assuming you're using the [default port](https://docs.taipy.io/en/latest/manuals/reference/taipy.gui.Gui/#taipy.gui.gui.Gui.run)), you'll see the following:
+
+![Defining a Multi-Page Application](multipage_application_7.png){width=50%}
+
+You're directed to the */home* URL automatically because it was the first page listed in the *pages* dictionary, and that's where you'll find our first page!
+
+If you change the URL from */home* to */temperature*, you will see the following:
+
+![Defining a Multi-Page Application](multipage_application_8.png){width=50%}
+
+## How to navigate between pages?
+
+Manually changing the URL to navigate between pages isn't ideal. 
+Taipy provides several methods for adding navigation to your multi-page application, 
+making it more user-friendly and intuitive.
+
+### 1. The navbar
+
+![The navbar](multipage_application_9.gif){width=50%}
+
+One straightforward method to make your application's navigation more appealing 
+is by utilizing the Taipy navbar control. 
+To incorporate the navbar into the home page, all you need to do is add a single line 
+to the beginning of the "home_md" page definition:
+
+![The navbar](multipage_application_9.png){width=100%}
+
+By default, the navbar control automatically creates an entry for each page in the *pages* dictionary, 
+requiring no further required specification of properties — making it a quick and easy way to add navigation.
+
+However, the code change above only added a navbar to the home_md page — we would still need to make the same 
+change to temperature_md and any other page we have in our application. 
+A better alternative that doesn’t require any code modification in any of the pages is to add a **root page**.
+
+Rather than modifying each page to include the navbar, we can also simply modify `main.py` to utilize the [root page](https://docs.taipy.io/en/latest/manuals/gui/pages/#root-page):
```

Comment:
```suggestion
Rather than modifying each page to include the navbar, we can also simply modify `main.py` to
utilize the [root page](../../../manuals/gui/pages/#root-page):
```